.github/labeler.yml

```diff
@@ -14,7 +14,7 @@ labels:
   - label: "dependencies"
     sync: true
     matcher:
-      title: "^build.*"
+      title: "^build: .*"
```

Comment:
I don't think you want to do this... it will miss `build(deps)` and `build(dev-deps)`... which is all the dependency stuff